src/compat/qsort.c

```diff
@@ -255,3 +258,10 @@ _quicksort (void *const pbase, size_t total_elems, size_t size,
   }
 }
 #endif /* !HAVE_QSORT_R_GLIBC */
+#else
+#if _MSC_VER
+#warning "C Preprocessor determined we will use system function qsort_s]"
```

Comment:
Replacing by
```
#pragma message("C Preprocessor determined we will use system function qsort_s]")
```
at least makes the compiling go silently through it.

Reply:
OK, now using pragma message